PHP/CodeSniffer/Standards/Sentact/ruleset.xml

```diff
@@ -2,10 +2,19 @@
 <ruleset name="Sentact">
     <description>The Sentact coding standard.</description>
 
-    <rule ref="PSR2"/>
+    <rule ref="PSR2">
+        <exclude name="PSR1.Classes.ClassDeclaration.MissingNamespace"/>
```

Comment:
Doesn't this remove the rule everywhere? We should have namespaces in normal files.

Reply:
it does. If you see an option to add criteria to this that would be better. Alternatively we just need to live with the errors or remove the error output in general

Reply:
Hmm. We definitely want namespaces in other files. I think it should be pretty hard to forget namespace if you're including the file anywhere else. Let's try it out and see how things go.